app/assets/stylesheets/custom.scss

```diff
@@ -291,3 +291,33 @@ ul, menu, dir {
 .filter {
   text-align: center;
 }
+
+
+// /* =========================
+//
+//      USER LISTINGS INDEX PAGE
+//
+//    =========================*/
+
+#img_container {
```

Comment:
Avoid using id selectors<br>Selector `img_container` should be written in lowercase with hyphens